src/tr2/game/shell/common.c

```diff
@@ -424,9 +424,13 @@ static void M_HandleConfigChange(const EVENT *const event, void *const data)
 
     if (CHANGED(visuals.fog_start) || CHANGED(visuals.fog_end)
         || CHANGED(visuals.water_color.g) || CHANGED(visuals.water_color.b)
-        || CHANGED(visuals.water_color.r)) {
+        || CHANGED(visuals.water_color.r) || CHANGED(visuals.use_psx_fov)) {
```

Comment:
Rather than doing it for this check can we bundle it with the biggest one that does the M_ResetRendererViewport?

Reply:
Thanks, updated.